src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -59,18 +58,11 @@
             "java", "nio", "zip", "net", "prefs", "jvm", "strictmath", "j2pkcs11", "sunec", "extnet", "libchelper"
     );
 
-    private static final List<String> linuxfxlibs = Arrays.asList( "-Wl,--whole-archive",
+    private static final List<String> linuxfxlibs = List.of(
+            "-Wl,--whole-archive",
             "-lprism_es2", "-lglass", "-lglassgtk3", "-ljavafx_font",
             "-ljavafx_font_freetype", "-ljavafx_font_pango", "-ljavafx_iio",
-            "-ljfxmedia", "-lfxplugins", "-lavplugin",
-            "-Wl,--no-whole-archive", "-lGL", "-lX11","-lgtk-3", "-lgdk-3",
-            "-lpangocairo-1.0", "-lpango-1.0", "-latk-1.0",
-            "-lcairo-gobject", "-lcairo", "-lgdk_pixbuf-2.0",
-            "-lgio-2.0", "-lgobject-2.0", "-lglib-2.0", "-lfreetype",
-            "-lpangoft2-1.0", "-lgstreamer-lite",
-            "-lgthread-2.0", "-lstdc++", "-lz", "-lXtst",
-            "-lavcodec", "-lavformat", "-lavutil",
-            "-lasound", "-lm", "-lgmodule-2.0"
+            "-ljfxmedia", "-lfxplugins", "-lavplugin"
```

Comment:
I think the -Wl,--no-whole-archive should be here, to clearly close the block of libs we need completely

Reply:
added